packages/block-editor/src/components/block-selection-clearer/index.js

```diff
@@ -19,20 +19,25 @@ export function useBlockSelectionClearer( ref ) {
 			return;
 		}
 
-		function onFocus() {
+		function onMouseDown( event ) {
+			// Only handle clicks on the canvas, not the content.
+			if ( event.target.closest( '.wp-block' ) ) {
```

Comment:
This condition could be removed eventually when the content is directly rendered into the wrapper the `mousedown` event is listened for. Currently there are a few wrappers in between the selection clearer and the block for which we also need to detect clicks.

Reply:
The  changes in this PR break the selection clearer in the widgets screen.  `.wp-block` must not be enough.

Ii also wonder why you're using mousedown, focusing outside (without  clicking) should also have   the same behavior right?

Reply:
I'm going to restore the previous  behavior for now to perform a patch release.